### ISSUE TITLE
SmartCity-Accessibility #123 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#105]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 245 lines...]
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[45,9] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[45,49] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[45,65] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[49,1] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[49,4] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[49,7] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[49,49] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[56,9] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[56,53] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[56,69] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[56,70] not a statement
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[56,71] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[59,1] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[59,3] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[59,5] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[60,17] > expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[60,23] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[60,40] ')' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[60,32] not a statement
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[60,42] not a statement
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[60,45] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[66,9] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[66,43] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[66,72] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[66,73] not a statement
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[66,76] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[67,1] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[67,3] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[67,5] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[67,7] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[68,17] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[72,9] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[72,41] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[72,74] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[72,75] not a statement
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[72,82] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[73,1] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[73,4] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[73,7] illegal start of expression
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[73,49] ';' expected
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java:[90,31] ';' expected
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Arthur Sapozhnikov] 1b58cf40a48feee5fc32b65fb38353de3da75416 - Spartanized a bit

- [Arthur Sapozhnikov] b84f4c5938881871e9327a3418fca9e32b247771 - Added pin and un-pin to Location

- [Arthur Sapozhnikov] a587b71ab8a15d7d3d731924388105c816fdebbc - Added mesaages in according to issue #104. closing #104

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/123/)

**Comment by kapalex** [comment]
@ArthurSap I have no idea what you did but you pretty much massacred the build

**Comment by ArthurSap** [comment]
I myself don't know, git started screaming something was wrong but It didnt say waht
I will try to fix it

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
